include/functions.php

```diff
@@ -172,3 +172,20 @@ function pll_set_constant( string $constant_name, $value ): bool {
 
 	return define( $constant_name, $value ); // phpcs:ignore WordPressVIPMinimum.Constants.ConstantString.NotCheckingConstantName
 }
+
+/**
+ * Determines whether a plugin is active.
```

Comment:
I propose to add a comment on why this function is useful:
```suggestion
 * Determines whether a plugin is active.
 *
 * We define our own function because `is_plugin_active()` is available only in the backend.
```